### PR TITLE
Feat/vav

### PR DESCRIPTION
**Pull request by aviveiros11**
Adds the NREL cooling only vav tasty validation test

### CODE REVIEW

**Comment by aviveiros11** [comment]
Werkin now, just need to replace the static namespace bug.  I will wait to merge this until that fix is on develop

**Comment by corymosiman12** [comment]
🌭  🌭  <-- that's a double hot dawg

File: tasty/source_shapes/haystack/nrel.json [hunk 1]
```diff
@@ -2,6 +2,123 @@
...
+    {
+      "name": "ZoneRelativeHumidityShape",
+      "types": ["air-humidity-sensor"],
+      "tags": ["zone"]
+    },
+    {
+      "name": "BinaryOccupancySensorShape",
+      "types": ["point"],
+      "tags": ["occupied", "sensor"],
+      "tags-custom": ["occupancyindicator"]
```
Comment (corymosiman12) [inline tasty/source_shapes/haystack/nrel.json]: `occupancyIndicator`

File: tasty/source_shapes/haystack/nrel.json [hunk 2]
```diff
@@ -28,6 +145,46 @@
           }
         ]
       }
+    },
+    {
+      "name": "NREL-VAV-SD-Cooling-Only-Shape",
+      "types": ["coolingOnly-vav"],
+      "predicates": {
+        "requires": [
```
Comment (corymosiman12) [inline tasty/source_shapes/haystack/nrel.json:160]: check back to the spreadsheet for what is required vs optional (column I)

File: .python-version [hunk 3]
```diff
@@ -0,0 +1 @@
+3.7.4
```
Comment (corymosiman12) [inline .python-version]: can you remove this file

File: tasty/source_shapes/haystack/nrel.json [hunk 4]
```diff
@@ -2,6 +2,123 @@
...
+    {
+      "name": "ZoneTemperatureCoolingStandbySetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "cooling", "standby"]
+    },
+    {
+      "name": "ZoneTemperatureHeatingStandbySetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "heating", "standby"]
+    },
```
Comment (corymosiman12) [inline tasty/source_shapes/haystack/nrel.json:60]: Can you create a new function grouping for this based on the spreadsheet? For example, use these six shapes to create a new (composite) shape called: `ZoneSetpointsThermalComfort`. Then the vav would look something like:
```
      "name": "NREL-VAV-SD-Cooling-Only-Shape",
      "types": ["coolingOnly-vav"],
      "shape-mixins": ["ZoneSetpointsThermalComfort", ...other functional groups... ]
}
```

Then this becomes really modular / reusable for unitary equipment / etc.

**Review by corymosiman12 — CHANGES_REQUESTED** [review]
dude looks great just minor changes on:
- required vs optional shapes on da box
- some tag names
- I just made the functional grouping comment on the one, but can you do that for like all of the functional systems Steve laid out?

File: tasty/source_shapes/haystack/nrel.json (hunk 2 above)
Comment (aviveiros11) [reply tasty/source_shapes/haystack/nrel.json:160]: 👍

**Comment by corymosiman12** [comment]
image

- Updated groupings
- Added missing

**Comment by aviveiros11** [comment]
@corymosiman12 I can confirm that tests pass on my end as expected 🚀

**Comment by corymosiman12** [comment]
perfecto! merging

**Review by corymosiman12 — APPROVED** [review]

**Pull request merged**